frontend/src/components/common/Table/Table.tsx

```diff
@@ -136,6 +145,8 @@ export default function Table<RowItem extends Record<string, any>>({
   filterFunction,
   errorMessage,
   loading,
+  noActions,
```

Comment:
I think we should call it the same as MRT: `enableRowActions` and have the same default: no actions by default.
Otherwise we're also breaking expecations as now all plugins using this table will have to set `noActions={true}` or the table will look different.